app/controllers/genres_controller.rb

```diff
@@ -0,0 +1,27 @@
+class GenresController < ApplicationController
+  before_action :set_genre, only: %i[show]
+
+  def index
+    @genre  = Genre.new
+    @genres = Genre.all
+  end
+
+  def show
+  end
+
+  def create
+    @genre = Genre.new(genre_params)
+    @genre.save
+    redirect_to genres_path
+  end
```

Comment:
What would you want to happen if the Genre doesn't save? Right now the user gets directed right back to the genre index without knowing their action didn't take place.

Reply:
Additionally, while the form is scoped to an admin, you'll also want to make sure that this method is scoped to an admin as well. Right now any user could make a POST to /create and create a record when we only want admin users to be able to do so.